tests/checkers/unittest_base_checker.py

```diff
@@ -87,9 +90,34 @@ def test_base_checker_doc() -> None:
 
 def test_base_checker_ordering() -> None:
     """Test ordering of checkers based on their __gt__ method."""
+    linter = PyLinter()
```

Comment:
```suggestion
def test_base_checker_ordering(linter: PyLinter) -> None:
    """Test ordering of checkers based on their __gt__ method."""
```
I think this work because the PyLinter is a fixture.